### ISSUE TITLE
a11y violation on ‘Landmarks should have a unique role or role/label/title’ on ‘section[aria-labelledby=“Live-event-1”]’

### ISSUE THREAD

**Issue by LilyL0u** [bbc/simorgh#9456]
Axe Core has flagged an issue where we have duplicate aria-labelledby attributes on home page sections. This violates the landmark-unique rule in Axe Core - https://dequeuniversity.com/rules/axe/4.1/landmark-unique

The attempted fix before was https://github.com/bbc/simorgh/issues/9415. However the violation is still being caught in the E2Es.

<img width="991" alt="Screenshot 2021-09-27 at 13 36 30" src="https://user-images.githubusercontent.com/54575620/134909291-c7344e41-e558-4a92-ab5a-fee5eea46dd6.png">

The same violation occurs on both canonical and amp on the Arabic front page.

Before the fix, we had two aria-labelledby="Live-event-1"
<img width="808" alt="Screenshot 2021-09-27 at 13 38 24" src="https://user-images.githubusercontent.com/54575620/134909554-c682cc5a-8466-431e-befa-5df2676aa1d4.png">

Now, this has been fixed so that there is only one aria-labelledby="Live-event-1", but the violation is still occurring.

<img width="1340" alt="Screenshot 2021-09-27 at 13 40 28" src="https://user-images.githubusercontent.com/54575620/134909840-986dc63e-7488-4094-a38b-f7f87b4352f4.png">

As you can see in the screenshot, the second aria-labelledby="Live-event-1" has disappeared. There is still an id="Live-event-1", but I don't think this would cause the violation?

I looked further into this and here are my thoughts so far:

When I run the axe tool on the page myself, it also finds the non unique landmark problem. It gives this detail to
<img width="789" alt="Screenshot 2021-09-24 at 12 19 27" src="https://user-images.githubusercontent.com/54575620/134910059-c46af0cf-d290-4ded-8971-66e2d0fc6349.png">
o

Ids on that are different though - even though they are live related.

These are the correct and incorrect aria-labelledby examples on the rule as shown in https://dequeuniversity.com/rules/axe/4.1/landmark-unique

Correct:
<img width="480" alt="Screenshot 2021-09-24 at 12 23 16" src="https://user-images.githubusercontent.com/54575620/134910225-69503086-3a4f-47ec-812d-bc8cf80a0807.png">

Incorrect:
<img width="513" alt="Screenshot 2021-09-24 at 12 23 23" src="https://user-images.githubusercontent.com/54575620/134910298-4d5dfcd5-188c-4c95-ab00-9e06439a271e.png">

In these the difference is that in the first and second lines, the correct one has also the non-id text different, and the incorrect one has the non-id text the same. So, if we can look at text, is it perhaps not liking that we have two sections with the title being ‘Live’?

In these screenshots are the two 'live' sections on the arabic front page (translated into English so you can see)

<img width="1336" alt="Screenshot 2021-09-27 at 13 45 22" src="https://user-images.githubusercontent.com/54575620/134910644-2d1ebcf2-357f-4d24-a73a-95e9a6942df3.png">

<img width="1335" alt="Screenshot 2021-09-27 at 13 50 05" src="https://user-images.githubusercontent.com/54575620/134911300-6adda072-1825-4f0c-867d-755bbbe4ac3f.png">

If this is the case, do we want to keep the rule and change the page - we cannot have two sections called 'Live'.
OR do we want to keep the page, and the allowance to have two sections with the same text, and make an exception in the rule?

**Comment by LilyL0u** [comment]
This has been raised in the ws editorial channel to see if the names of the sections can be changed

**Comment by LilyL0u** [comment]
Conclusion, editorial changed the section titles so they are no longer the same. The E2Es are now passing.
